ideascube/serveradmin/wifi.py

```diff
@@ -168,7 +168,12 @@ def all(cls):
         result = []
 
         for connection in NMSettings.ListConnections():
-            if '802-11-wireless' not in connection.GetSettings():
+            try:
+                if '802-11-wireless' not in connection.GetSettings():
+                    continue
+            except DBusException as e:
+                # NM throws an exception if we try to use GetSettings on a
+                # connection owned by another user. We just ignore it.
                 continue
 
```

Comment:
Sorry but this is a bit difficult to read.
It could be better to first get the setting in the try/except and after test the value of the setting.